integration/skaffold/helper.go

```diff
@@ -249,6 +249,21 @@ func (b *RunBuilder) Run(t *testing.T) error {
 	return nil
 }
 
+// RunWithProcess starts the skaffold command and returns the process id and error.
+func (b *RunBuilder) RunWithProcess(t *testing.T) (*os.Process, error) {
+	t.Helper()
+
+	cmd := b.cmd(context.Background())
+	logrus.Infof("Running %s in %s", cmd.Args, cmd.Dir)
+
+	start := time.Now()
+	if err := cmd.Start(); err != nil {
+		return nil, fmt.Errorf("skaffold %q: %w", b.command, err)
+	}
+	logrus.Infof("Ran %s in %v", cmd.Args, util.ShowHumanizeTime(time.Since(start)))
```

Comment:
`cmd.Start` returns immediately, so I think this log line is incorrect.

Reply:
ah yeah good point, I copied this from the other functions. I'll just remove it, it's not really relevant here